quipucords/quipucords/tests_feature_flags.py

```diff
@@ -0,0 +1,154 @@
+        assert feature_name not in dict_with_test_flags
+
+
+@pytest.mark.parametrize(
+    'env_name,env_value,feature_name',
+    (('TEST_QPC_FEATURE_', '1', 'TEST'),
+     ('TEST_QPC_FEATURE_', '1', 'TEST_'),
+     ('QPC_TEST1_FEATURE_', '0', 'TEST1'),
+     ('QPC_TEST1_FEATURE_', '0', '_TEST1'),
+     ('QPC_TEST1_FEATURE_', '0', 'TEST1_')
+     )
+)
+def test_function_only_adds_names_follow_standard(
+        env_name, env_value, feature_name):
+    """Tests if function only adds variables that start with QPC_FEATURE_."""
+    with mock.patch.dict(os.environ, ({env_name: env_value})):
+        dict_with_test_flags = get_feature_flags_from_env()
+        assert feature_name not in dict_with_test_flags
+
+
+@pytest.mark.parametrize(
+    'env_name,env_value,feature_name,feature_value',
+    (('qpc_feature_test', '1', 'TEST', True),
+     ('qpc_feature_TEST1', '0', 'TEST1', False),
+     ('QPC_feature_TEST2', '0', 'TEST2', False),
+     ('qpc_FEATURE_test3', '1', 'TEST3', True)
+     )
+)
+def test_if_function_is_not_case_sensitive(env_name, env_value, feature_name,
+                                           feature_value):
+    """Tests if function is not case-sensitive and env names are stored following.
+
+    the standard.
+    """
+    with mock.patch.dict(os.environ, ({env_name: env_value})):
+        dict_with_test_flags = get_feature_flags_from_env()
+        assert feature_name in dict_with_test_flags
+        assert dict_with_test_flags[feature_name] is feature_value
+
+
+@pytest.fixture
+def setup_feature_flag_instance_for_tests():
+    """Set up instance of FeatureFlag class for tests."""
+    with mock.patch.dict(os.environ, {'QPC_FEATURE_TEST': '1'}):
+        dict_with_test_flags = get_feature_flags_from_env()
+        feature_flag_instance = FeatureFlag(dict_with_test_flags)
+        return feature_flag_instance
+
+
+def test_if_instance_contains_all_attributes(
+        setup_feature_flag_instance_for_tests):
+    """Tests if the constructor loads all attributes correctly."""
+    assert hasattr(setup_feature_flag_instance_for_tests, 'TEST')
+    assert hasattr(setup_feature_flag_instance_for_tests, 'OVERALL_STATUS')
+
+
+def test_if_instance_attributes_values_are_correct(
+        setup_feature_flag_instance_for_tests):
+    """Tests if the right values are attributed to attribute."""
+    assert setup_feature_flag_instance_for_tests.TEST is True
+    assert setup_feature_flag_instance_for_tests.OVERALL_STATUS is False
+
+
+def test_is_feature_active(setup_feature_flag_instance_for_tests):
+    """Tests method is_feature_active."""
+    assert setup_feature_flag_instance_for_tests.is_feature_active(
+        'TEST') is True
+    assert setup_feature_flag_instance_for_tests.is_feature_active(
+        'OVERALL_STATUS') is False
+    with pytest.raises(ValueError):
+        assert setup_feature_flag_instance_for_tests.is_feature_active(
+            'FALSE_ATTRIBUTE') is True
```

Comment:
This assertion will never run if `ValueError` is properly raised.
```suggestion
        setup_feature_flag_instance_for_tests.is_feature_active('FALSE_ATTRIBUTE')
```